cmd/helmer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.13 as builder
+COPY . /go/src/github.com/kubeapps/kubeapps
+WORKDIR /go/src/github.com/kubeapps/kubeapps
+
+ARG VERSION
+RUN CGO_ENABLED=0 go build -a -installsuffix cgo -ldflags "-X main.version=$VERSION" ./cmd/helmer
+
+FROM scratch
+COPY --from=builder /go/src/github.com/kubeapps/kubeapps/helmer /agent
```

Comment:
I've read that using the `USER` directive in every Dockerfile is recommended. I have not had time to insert `USER nobody` in this file and test it, just writing this as a reminder.